src/main.jsx

```diff
@@ -7,45 +7,56 @@ import { render } from 'react-dom'
 import { Router, Route, Redirect, hashHistory } from 'react-router'
 import { I18n } from './plugins/preact-polyglot'
 
+import SettingsStore from './store/SettingsStore'
+import Provider from './store/Provider'
+
 import App from './components/App'
+import AccountManagement from './containers/AccountManagement'
 
 const context = window.context
 const lang = document.documentElement.getAttribute('lang') || 'en'
 
+const stackDomain = 'http://cozy.local:8080'
+
+// store
+const store = new SettingsStore(stackDomain)
+
 document.addEventListener('DOMContentLoaded', () => {
   render((
-    <I18n context={context} lang={lang}>
-      <Router history={hashHistory}>
-        <Route component={(props) =>
-          <App {...props} />}
-        >
-          <Redirect from='/' to='account' />
-          <Route
-            path='account'
-            component={(props) =>
-              <p>Account</p>
-            }
-          />
-          <Route
-            path='connectedDevices'
-            component={(props) =>
-              <p>Connected devices</p>
-            }
-          />
-          <Route
-            path='storage'
-            component={(props) =>
-              <p>Storage</p>
-            }
-          />
-          <Route
-            path='emailNotifications'
-            component={(props) =>
-              <p>Email Notifications</p>
-            }
-          />
-        </Route>
-      </Router>
-    </I18n>
+    <Provider store={store}>
+      <I18n context={context} lang={lang}>
+        <Router history={hashHistory}>
+          <Route component={(props) =>
+            <App {...props} />}
+          >
+            <Redirect from='/' to='account' />
+            <Route
+              path='account'
+              component={(props) =>
```

Comment:
You can simply write `<Route path="account" component={AccountManagement} />` here. The usage of the `component={props => <AccountManagement {...props} />}` syntax should be restricted to edge cases.